### PR TITLE
Merge 1.6 into master

### PR DESCRIPTION
**Pull request by ahocquard**
**Description (for Contributor and Core Developer)**

The last two commits fix :
- PIM-5862 : fix the bug differently on master due to a complete rewriting of the views in the datagrid 

- Fix an issue with the standard format for ODM. Families were not correctly normalized in MongoDB. This bug occurs because we did not have any behat to detect that before this merge.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       |
| Added Behats                      |
| Changelog updated                 |
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  |
| Tech Doc                          |

### CODE REVIEW

File: features/Context/Page/Base/Grid.php [hunk 1]
```diff
@@ -154,7 +154,7 @@ public function __construct($session, $pageFactory, $parameters = [])
                 'Filters'               => ['css' => '.filter-box, .filter-wrapper'],
                 'Grid toolbar'          => ['css' => '.AknGridToolbar'],
                 'Manage filters'        => ['css' => 'div.filter-list'],
-                'Configure columns'     => ['css' => 'a:contains("Columns")'],
+                'Configure columns'     => ['css' => '#configure-columns'],
```
Comment (damien-carcel) [inline features/Context/Page/Base/Grid.php]: Is there no way to use a class rather than an ID?

File: features/datagrid/datagrid_views.feature [hunk 2]
```diff
@@ -137,9 +144,43 @@ Feature: Datagrid views
...
+    And the row "black-boots" should contain:
+      | column | value       |
+      | Name   | Black boots |
+      | Family | Boots       |
+
   Scenario: Successfully change grid channel
-    Given I should see the text "Tablet"
-    And I filter by "scope" with operator "" and value "Mobile"
+    Given I am on the products page
+    Then I should see the text "Tablet"
```
Comment (damien-carcel) [inline features/datagrid/datagrid_views.feature]: And

File: features/enrich/variant-group/edit_variant_group_attribute_value.feature [hunk 3]
```diff
@@ -110,6 +112,33 @@ Feature: Editing attribute values of a variant group also updates products
...
+      | green |
+    When I am on the "caterpillar_boots" variant group page
+    And I visit the "Attributes" tab
+    And I visit the "Marketing" group
+    And I add available attributes Simple
+    And I change the "Simple" to "red"
+    And I save the variant group
+    And I wait for the options to load
+    And I switch the locale to "fr_FR"
+    When I change the "Simple" to "blue"
```
Comment (damien-carcel) [inline features/enrich/variant-group/edit_variant_group_attribute_value.feature]: And, or add a Then somewhere before ;)

File: features/mass-action/edit_common_attributes-01.feature [hunk 4]
```diff
@@ -152,6 +152,18 @@ Feature: Edit common attributes of many products at once
...
+  @jira https://akeneo.atlassian.net/browse/PIM-6008
+  Scenario: Successfully mass edit scoped product values with special chars
+    Given I set product "pump" family to "boots"
+    When I mass-edit products boots and pump
+    And I choose the "Edit common attributes" operation
+    And I display the Description attribute
+    And I change the Description to "&$@(B°ar'<"
+    And I move on to the next step
+    And I wait for the "edit-common-attributes" mass-edit job to finish
+    And the english tablet Description of "boots" should be "&$@(B°ar'<"
```
Comment (damien-carcel) [inline features/mass-action/edit_common_attributes-01.feature]: Then

File: features/user/create_edit_and_delete_role.feature [hunk 5]
```diff
@@ -39,3 +39,8 @@ Feature: Create and delete a user role
       | Role |  |
     When I save the role
     Then I should see validation tooltip "This value should not be blank."
+    When I edit the "Administrator" role
+    And I fill in the following information:
+      | Role | ThisIsARoleLabelWith27Chars |
+    When I save the role
```
Comment (damien-carcel) [inline features/user/create_edit_and_delete_role.feature]: And

File: features/datagrid/datagrid_views.feature (hunk 2 above)
Comment (anaelChardan) [inline features/datagrid/datagrid_views.feature]: I should see is always with a then as it is an assertion no?

File: features/enrich/variant-group/edit_variant_group_attribute_value.feature (hunk 3 above)
Comment (anaelChardan) [inline features/enrich/variant-group/edit_variant_group_attribute_value.feature]: +1

File: features/mass-action/edit_common_attributes-01.feature (hunk 4 above)
Comment (anaelChardan) [inline features/mass-action/edit_common_attributes-01.feature]: +1

File: features/Context/Page/Base/Grid.php (hunk 1 above)
Comment (damien-carcel) [inline features/Context/Page/Base/Grid.php]: To bad

**Comment by anaelChardan** [comment]
o/ :)

**Pull request merged**